config/locales/links/fr.yml

```diff
@@ -16,12 +16,12 @@ fr:
     go_back: Retourner
     help: Aide
     new_tab: '(ouvre un nouvel onglet)'
-    next: Suivant
     passwords:
       forgot: Vous avez oublié votre mot de passe?
     privacy_policy: Confidentialité et sécurité
     resend: Renvoyer
     reverify: Veuillez vérifier votre identité de nouveau.
+    sign_in: Connexion
     sign_out: Déconnexion
```

Comment:
We should probably split this out to a separate ticket, but based on the prior conversation about verb forms of the translation, I think we'd want to revisit this one, since this reads to me as a noun.